### PR TITLE
Update README.MD to improve callback docs

### PR DESCRIPTION
**Pull request by axis80**
Changes docs to reflect new name of signInSuccessWithAuthResult callback, and adds a mention of the error callback.

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -300,16 +300,22 @@ private firebaseAuthChangeListener(response) {
...
 
 ```html
-<firebase-ui (signInSuccess)="successCallback($event)"></firebase-ui>
+<firebase-ui
+    (signInSuccessWithAuthResult)="successCallback($event)"
+    (signInFailure)="errorCallback($event)"></firebase-ui>
 ```
 
 ```typescript
 successCallback(signInSuccessData: FirebaseUISignInSuccess) {
```
Comment (RaphaelJenni) [inline README.MD]: Could you replace the `FirebaseUISignInSuccess` with `FirebaseUISignInSuccessWithAuthResult`

**Comment by RaphaelJenni** [comment]
First of all, thanks allot! I added a little comment. If that is fixed, it looks good to be merged.

**Comment by axis80** [comment]
OK, I think I made that edit correctly. Let me know if any other changes are needed!

**Comment by RaphaelJenni** [comment]
Thanks allot!

**Pull request merged**